templates/_layout.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>404: Page not found</title>
```

Comment:
Doesn't matter much for now, but I guess layout shouldn't have a hardcoded '404' title.